apiserver/facades/agent/keyupdater/facade_mock_test.go

```diff
@@ -126,6 +126,44 @@ func (c *MockModelContextClockCall) DoAndReturn(f func() clock.Clock) *MockModel
 	return c
 }
 
+// ControllerModelUUID mocks base method.
+func (m *MockModelContext) ControllerModelUUID() model.UUID {
```

Comment:
Isn't there a central facade mock?

Reply:
I'd probably state that you should even know about the facade context. There is a modelling issue here.